client/app/billing/confirmTerminate/billing-confirmTerminateLegacy.service.js

```diff
@@ -0,0 +1,59 @@
+export default class {
+  constructor($q, OvhHttp) {
+    this.$q = $q;
+    this.OvhHttp = OvhHttp;
+  }
+
+  static getServiceTypeFromPrefix(serviceApiPrefix) {
+    return serviceApiPrefix
```

Comment:
Could you please provide comments about what this does ?

```js
// Turns "hello you" to "h3ll0 y0u"
```

so we can understand easier what it does ? :)